test/integration/other/encryption.js

```diff
@@ -355,7 +355,7 @@ describe('managed encryption', () => {
                 result['media/testfile.jpg'].should.equal('hello this is a suffixed file');
               }))))));
 
-    it('should decrypt client audit log attachments', testService((service, container) =>
+    it('should decrypt client audit log attachments (1)', testService((service, container) =>
```

Comment:
This test looks like it's running `exhaust(container)` to process the client audits and that they are processed successfully without error. But since they are encrypted, the worker does not actually decrypt them, so the `client_audits` table has 0 rows.

Maybe a better name for the test is `should successfully process encrypted client audit log attachments without decrypting`?

Reply:
Thanks for closer reading of this PR!

> `should successfully process encrypted client audit log`

Does it actually process them?!  I'd expect entries in the `client_audits` table, but the test asserts there are none... Would a better title be: `should mark encrypted client audit log attachments as processed, but not process them`?

Reply:
Something like that! In `lib/worker/submission.attachment.update.js` I think it's skipping attachments of encrypted submissions. So it's "processing" the attachment event to check for client audits but actually ignoring them.

Reply:
`should skip encrypted client audit log attachments and mark them as processed`?